docs/content/en/docs/getting-started/observability/_index.md

```diff
@@ -1,6 +1,280 @@
 weight: 45
 ---
 
+The Keptn Lifecycle Toolkit (KLT) makes any Kubernetes deployment observable.
+You can readily see why a deployment takes so long or why it fails,
+even when using multiple deployment tools.
+Keptn introduces a concept of an application
+which is an abstraction that connects multiple
+Workloads belonging together.
+In other words, KLT, creates a distributed end-to-end trace
+of everything Kubernetes does in the context of a Deployment.
+
+The observability data is an amalgamation of the following:
+
+- DORA metrics are collected out of the box
+  when the Lifecycle Toolkit is enabled
+- OpenTelemetry runs traces that show everything that happens in the Kubernetes cluster
+  and can display this information with dashboard tools
+  such as Grafana.
+- Specific metrics that you can define to monitor
+  information from all the data providers configured in your cluster.
+
+The Keptn Lifecycle Toolkit can provide this information
+for all applications running in your cluster,
+even if they are using different deployment tools.
+And it can capture metrics from multiple data sources
+using multiple data platforms.
+With KLT deployed on your cluster,
+you can easily monitor what is happening during a deployment into your Kuberenetes cluster,
+and quickly get data to help you understand issues such as
+why a deployment took so long or why it failed.
+
+## Using this exercise
+
+This exercise shows how to standardize access
+to the observability data for your cluster.
+It is based on the
+[simplenode-dev](https://github.com/keptn-sandbox/klt-on-k3s-with-argocd)
+example.
+You can clone that repo to access it locally
+or just look at it for examples
+as you implement the functionality "from scratch"
+on your local Kubernetes deployment cluster.
+The
+[README](https://github.com/keptn-sandbox/klt-on-k3s-with-argocd/blob/main/setup/observability/README.md)
+file for that repo contains useful information.
+
+Two videos are available
+to walk you through this exercise if you prefer:
+
+- [Introducing Keptn Lifecycle Toolkit](https://youtu.be/449HAFYkUlY)
+- [Use SLOs and get DORA the Native K8s way!](https://www.youtube.com/watch?v=zeEC0475SOU)
+
+In the
+[Getting started with Keptn metrics](../metrics)
+exercise, you learn how to define and use Keptn metrics.
+You may want to complete that exercise before doing this exercise
+although that is not required.
+
+This exercise shows how to standardize access
+to the observability data for your cluster.
+The steps are:
+
+1. [Install and enable]( #install-and-enable-klt)
+   the Lifecycle Toolkit on your cluster
+1. [Integrate the Lifecycle Toolkit with your applications](#integrate-the-lifecycle-toolkit-with-your-applications)
+1. [DORA metrics](#dora-metrics)
+1. [Using OpenTelemetry](#using-opentelemetry)
+1. [Keptn metrics](#keptn-metrics)
+1. [View the results](#view-the-results)
+
+## Install and enable KLT
+
+To install and enable the Keptn Lifecycle Toolkit on your cluster:
```

Comment:
One user was asking us whether the KLT can be installed in a separate cluster, unsure if this is the right spot to stress it out or if we should do it in the installation. But I would have the info that KLT communicates with the k8s scheduler to stop deployment from proceeding until tasks/evaluations pass, thus it needs to be installed in the cluster you want to monitor.

Reply:
I saw that question and I agree -- it doesn't belong in the Getting Started Guide but in the main Installation section.  I will do a separate PR for this.